politeiad/backendv2/tstorebe/plugins/pi/hooks.go

```diff
@@ -140,6 +141,50 @@ func (p *piPlugin) proposalNameIsValid(name string) bool {
 	return p.proposalNameRegexp.MatchString(name)
 }
 
+// proposalStartDateIsValid returns whether the provided start date is valid.
+//
+// A valid start date of a proposal must be in the future.
+func (p *piPlugin) proposalStartDateIsValid(sd int64) bool {
+	return sd > time.Now().Unix()
+}
+
+// proposalEndDateIsValid returns whether the provided end date is valid.
+//
+// A valid end date must be after the start date and before the end of the
+// time interval set by the proposalEndDateMax plugin setting.
+func (p *piPlugin) proposalEndDateIsValid(sd int64, ed int64) bool {
```

Comment:
Variable abbreviations like this are good for structs because you can see the full struct definition and what it represents so you don't need long variable names, but doing this for non structs like you have here can make the code hard to read.  Would prefer a non-abbreviated version.

Either of these would be fine.

```suggestion
func (p *piPlugin) proposalEndDateIsValid(start int64, end int64) bool {
```
```suggestion
func (p *piPlugin) proposalEndDateIsValid(startDate int64, endDate int64) bool {
```

Reply:
Will keep that in mind 👍🏽